### PR TITLE
Add changelog entry for sqlalchemy pin

### PR DESCRIPTION
**Pull request by ryuusama09**
Fixes https://github.com/sunpy/sunpy/issues/6698

### CODE REVIEW

File: changelog/#6694 .bug.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+the current version of ``sqlalchemy`` has been pinned < ``2.0`` till investigation is being done on new
```
Comment (nabobalis) [inline changelog/#6694 .bug.rst:1]: The filename should have no spaces in it, nor a #.

For now, I think a trivial category would be enough.

File: changelog/6694 .bug.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+the current version of ``sqlalchemy`` has been pinned < ``2.0`` till investigation is being done on new
+release of version ``2.0`` so that it does not break the current database implementation
```
Comment (nabobalis) [inline changelog/6694 .bug.rst:2]:
```suggestion
The current version of ``SQLAlchemy`` has been fixed to be <2.0 until we have ensured that the current database module works with it.
```

**Comment by ryuusama09** [comment]
pre-commit.ci autofix

File: changelog/6694 .bug.rst (hunk 2 above)
Comment (ryuusama09) [reply changelog/6694 .bug.rst]: hopefully its ok now ? @nabobalis

Comment (nabobalis) [reply changelog/6694 .bug.rst]: my suggestion didn't make it in.

Comment (ryuusama09) [reply changelog/6694 .bug.rst]: implemented your comment

**Review by nabobalis — APPROVED** [review]

**Comment by nabobalis** [comment]
Thanks for the pull request @ryuusama09

**Comment by ryuusama09** [comment]
> Thanks for the pull request @ryuusama09

@nabobalis , will sunpy take part in gsoc 2023 ?

**Comment by nabobalis** [comment]
> > Thanks for the pull request @ryuusama09
> 
> @nabobalis , will sunpy take part in gsoc 2023 ?

Yes, we will be applying as an organization for 2023 is my understanding.

**Comment by ryuusama09** [comment]
> 
Great . I am highly interested to work more on this project. Can you suggest some prerequisites needed so that something big can be implemented :). Hope i can offer more to thhis organization

**Comment by nabobalis** [comment]
> Great . I am highly interested to work more on this project. Can you suggest some prerequisites needed so that something big can be implemented :). Hope i can offer more to this organization

What project do you prefer to, sunpy? 

I would suggest that you read over https://docs.sunpy.org/en/latest/dev_guide/contents/newcomers.html and then as you have been doing, tackle open issues that interest you.

Our GSoC info is found: https://github.com/sunpy/sunpy/wiki/Google-Summer-of-Code

**Comment by ryuusama09** [comment]
> > Great . I am highly interested to work more on this project. Can you suggest some prerequisites needed so that something big can be implemented :). Hope i can offer more to this organization
> 
> What project do you prefer to, sunpy?
> 
> I would suggest that you read over https://docs.sunpy.org/en/latest/dev_guide/contents/newcomers.html and then as you have been doing, tackle open issues that interest you.
> 
> Our general GSoC info is found: https://github.com/sunpy/sunpy/wiki/Google-Summer-of-Code It has not been updated for this year yet.

slr. 
If the projects list on gsoc page is not outdated , then i would like to work on a sunpy project

**Comment by nabobalis** [comment]
> slr.

Please don't call me sir.

> If the projects list on gsoc page is not outdated , then i would like to work on a sunpy project

The projects page has not been updated from 2022, that is currently in progress.

You do not start working on a GSoC project until you have been accepted. In the meantime,

I would suggest that you read over [docs.sunpy.org/en/latest/dev_guide/contents/newcomers.html](https://docs.sunpy.org/en/latest/dev_guide/contents/newcomers.html) and then as you have been doing, tackle open issues that interest you.

Our general GSoC info is found: [sunpy/sunpy/wiki/Google-Summer-of-Code](https://github.com/sunpy/sunpy/wiki/Google-Summer-of-Code)
It has not been updated for this year yet.

**Comment by nabobalis** [comment]
> I didn't call you sir , it is "slr" = sorry for late reply

I had no idea, I apologize.

**Pull request merged**